docs/config.rst

```diff
@@ -241,4 +241,4 @@ Environment Variables
 
 .. describe:: SENTRY_ENVIRONMENT
 
-    Optionally set the environment name, e.g. "staging", "production".
+    Optionally set the environment name, e.g. "staging", "production". You can also use `NODE_ENV` variable instead.
```

Comment:
Should probably change this to say, "Sentry will default to the value of `NODE_ENV`, if present."